trymito.io/pages/blog/[slug].tsx

```diff
@@ -2,87 +2,146 @@
+                </p>
+                {publishedAt && <p className={postStyles.post_additional_info}> {publishedAt}</p>}
+                {readingTime && <p className={postStyles.post_additional_info}>{readingTime} min read</p>}
               </div>
+              <h1>{props.post.title}</h1>
             </div>
+          </div>
 
-            <section className={pageStyles.background_card}>
-              <DownloadCTACard />
-            </section>
-          </main>
+          {/* Table of Contents, post contents, and CTA */}
+          <div className={postStyles.post_content_container}>
+            {/* Table of Contents */}
+            <div className={postStyles.post_toc}>
+              <PageTOC />
+            </div>
+            {/* Post Contents */}
+            <div className={postStyles.post_content}> 
+              <div dangerouslySetInnerHTML={{ __html: props.post.html }}/>
+              <div className={postStyles.share_section}>
+                <a className={postStyles.tweet_button}
+                  href={`https://twitter.com/intent/tweet?text=${encodeURIComponent(props.post.title ?? '')}&url=${encodeURIComponent(currentURL ?? '')}`}>
+                  <TwitterLogo />
+                </a>
+                <a className={postStyles.email_button}
+                  href={`mailto:?subject=${encodeURIComponent(props.post.title ?? '')}&body=${encodeURIComponent(currentURL ?? '')}`}>
+                    <EmailIcon />
+                  </a>
+                <a className={postStyles.linkedin_button}
+                  href={`https://linkedin.com/shareArticle?title=${encodeURIComponent(props.post.title ?? '')}&url=${encodeURIComponent(currentURL ?? '')}`}>
+                    <LinkedinLogo />
+                  </a>
+              </div>
 
-          <Footer />
+              {/* Suggested Posts */}
+              <div className={postStyles.suggested_posts_section}>
+                <h1>More Like This</h1>
+                <div className={postStyles.suggested_posts_container}>
+                  <Link href='/blog/automating-spreadsheets-with-python-101'>
+                    <div className={postStyles.suggested_post}>
+                      <h4>Automating Spreadsheets with Python 101</h4>
+                      <p>How to tell the difference between a good and bad Python automation target.</p>
+                    </div>
+                  </Link>
+                  <Link href='/blog/10-mistakes-to-look-out-for-when-transitioning-from-excel-to-python'>
+                    <div className={postStyles.suggested_post}>
+                      <h4>10 Mistakes To Look Out For When Transitioning from Excel To Python</h4>
+                      <p>10 Common Mistakes for new programmers transitioning from Excel to Python</p>
+                    </div>
+                  </Link>
+                  <Link href='/blog/quantifying-mitos-impact-on-analyst-python-productivity'>
```

Comment:
Can we update the design of these a bit to have some box around each one. Just to make it clear that there are four things here that you can click on? 

<img width="1028" alt="Screenshot 2024-02-19 at 10 26 31 AM" src="https://github.com/mito-ds/mito/assets/18709905/a94d5950-3e65-45f0-b382-58680ccf7ad6">

Reply:
Yeah I think that does a good job of making them look separate from each other. Maybe when we take another pass at the `trymito.io/blog` page we can update this with a similar style. But that's for later!